WcaOnRails/lib/country_city_validators/gb_city_validator.rb

```diff
@@ -92,6 +92,17 @@
   Wiltshire
   Worcestershire
   Yorkshire
+  Akrotiri\ and\ Dhekelia
+  Anguilla
+  Bermuda
+  British\ Virgin\ Islands
+  Cayman\ Islands
+  Falkland\ Islands
+  Gibraltar
+  Montserrat
+  Pitcairn\ Islands
+  Saint\ Helena\,\ Ascension\ and\ Tristan\ da\ Cunha
```

Comment:
I think the `\` here will be turned into a literal backslash character, which is not what we want. Proof:

```ruby
$ irb
irb(main):004:0> puts %w(Saint\ Helena\,\ Ascension\ and\ Tristan\ da\ Cunha)[0]
Saint Helena\, Ascension and Tristan da Cunha
=> nil
```

Note that a `\` got printed here.

Reply:
Oops yeah, I wasn't sure whether a comma would affect anything in Ruby, so I'll remove the \!